### ISSUE TITLE
different level increase rates for different mob types

### ISSUE THREAD

**Issue by lokka30** [lokka30/LevelledMobs#98]
Suggested by Oathkeeper

```
custom_mob_level:
    BABY_ZOMBIE:
        movement-speed: X
        max-health: X
```

etc. to override the default mob leveling amounts. Many different applications, though my main reason is that as my server stands everything is pretty balanced.. Except for the baby zombies and other baby version of mobs which become "the flash" as my players descript when they get to the higher levels, making them near impossible to kill. Being able to slightly nerf that one mob with different values would fix the problem, plus like I said, add a useful feature for all situations.

**Comment by stumper66** [comment]
This has been somewhat implemented in 2.0.  You can set levels for specific mobs including baby zombies.
To prevent baby zombies from becoming "the flash" you can limit them to level 1.

**Comment by stumper66** [comment]
Implemented in 2.2.0
